features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -577,6 +577,8 @@
   "authentication.adaptive.long_wait.page_refresh_interval": "500ms",
   "authentication.adaptive.long_wait.timout": "10s",
   "authentication.adaptive.long_wait.prompt": false,
+  "authentication.adaptive.execution_supervisor.thread_count": 1,
+  "authentication.adaptive.execution_supervisor.timeout": 500,
```

Comment:
let's have an entry in `org.wso2.carbon.identity.core.server.feature.unit-resolve.json` to specify that this config is in `ms`.

Reply:
added